### PR TITLE
Upgrade json to fix CVE + Upgrade Awestruct

### PR DESCRIPTION
**Pull request by ge0ffrey**
@triceo This also reverts some of the changes that you did earlier, which caused warning messages with me (and I believe Radovan and Jiri saw those showing up too with that change). However, I believe if you do a few `sudo gem pristine ... --version ...` commands on your machine, this will fix it. Would you mind pulling this PR locally and verifying that? I don't want to merge if we don't get it working on all our machines, including yours.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -39,6 +39,18 @@ And publish your changes (if you got authorization, otherwise Jenkins will do it
...
 ```
 
+==== Troubleshooting:
+
+* Error message `Could not find ffi-1.13.1 in any of the sources`
++
+If you get that error message, run:
++
+```
+sudo gem pristine ffi --version 1.13.1
```
Comment (ge0ffrey) [inline README.adoc:49]: After the awestruct upgrade, it didn't just work for me. I had to run this command (and the rake setup below it).

File: Gemfile [hunk 2]
```diff
@@ -1,9 +1,7 @@
 source 'https://rubygems.org'
-ruby '>= 2.4.0'
+ruby '>= 2.6.0'
```
Comment (ge0ffrey) [inline Gemfile:2]: @mbiarnes I hope this isn't a problem for our CI publishing?

**Review by triceo — APPROVED** [review]
Works for me.

**Review by rsynek — APPROVED** [review]
Works for me too, thanks!

**Comment by mbiarnes** [comment]
@ge0ffrey 
1. sudo rake setup worked properly
2. rake clean build worked properly
3. rake --dry-run publish (without any errors) 
LGTM

**Review by mbiarnes — APPROVED** [review]
+1

**Pull request merged**